ProjectLighthouse.Servers.GameServer/Controllers/MessageController.cs

```diff
@@ -55,16 +55,30 @@ public async Task<IActionResult> Announce()
     {
         GameTokenEntity token = this.GetToken();
 
-        string username = await this.database.UsernameFromGameToken(token);
+        UserEntity? user = await this.database.UserFromGameToken(token);
```

Comment:
```suggestion
        UserEntity? user = await this.database.UserFromGameToken(token);
        if (user == null) return this.BadRequest();
```

Don't commit this directly, implement this as needed in your editor